pkg/pods/manifest.go

```diff
@@ -41,6 +41,14 @@ type Manifest struct {
 	signature []byte
 }
 
+type JSONManifest struct {
+	// JSON encodable version of Manifest
+	Id                string                      `json:"id"`
+	Path              string                      `json:"path"`
```

Comment:
I created this struct for p2-show-health because it was failing to encode the pods.Manifest struct